debian/rules

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/make -f
 
+export DEB_BUILD_MAINT_OPTIONS=hardening=+all
+
 %:
```

Comment:
Well, looking at the list:
- HARDENING_FORMAT, we are already compile with -Wformat=2, so this does nothing
- FORTIFY - This  probably does nothing on this tree, but is performance-neutral
- STACKPROTECTOR - this has a runtime cost
- PIE - We don't have any performance critical executables, and libraries are already PIC, so this is neutral - however we should check that this flag is passed around properly in cmake, it is a bit tricky..
- RELRO - No performance cost for us
- BINDNOW - No performance cost for us

I looked into it some more and FC25 is enabling -fstack-protector-strong by default now. As is Centos 7 - so I think it is safe for debian to do it as well. If someone determines it is a problem we will have to handle it globally...

Can you check that pie worked properly?